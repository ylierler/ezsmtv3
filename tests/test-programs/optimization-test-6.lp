{a}.
b:-a.
c:-not a.
:-a, &sum{x}>=12.    
:~ a. [-1@1]
:~ &sum{x}=12. [-2@1]