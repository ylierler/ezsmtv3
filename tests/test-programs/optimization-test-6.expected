c 
x=0 
 0

b a 
x=(- 1) 
 -1

c 
x=12 
 -2